swap_meet/vendor.py

```diff
@@ -0,0 +1,97 @@
+from swap_meet.item import Item
+
+class Vendor:
+
+    def __init__(self, inventory = None):
+        if inventory is None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
```

Comment:
👍 this time the if statement is necessary! we shouldn't use lists and dictionaries as default parameters because then *all* instances of Vendor would use the same list or dictionary in our memory space.

But with the if-else statement, we an make sure that each instance gets its own unique and independent list! perfect!